src/perf-tests/README.md

```diff
@@ -0,0 +1,129 @@
+# Overview
+
+This perf-test stack uses [Gatling](https://gatling.io/), open source tool for load testing. The stack creates AWS Fargate Cluster to run the setup and Gatling scripts.
+
+The setup scripts found under [mock-scripts](./mock-scripts) folder
+- creates test users in Amazon Cognito userpool
+- load mock flight data into Flights table
+- zips Gatling reports and upload to S3 bucket
+
+The Gatling simulation script uses constantUsersPerSec and rampUsersPerSec to inject users for the given scenarios. By default, it uses `5 users` for a duration of `300 seconds` OR `5 minutes`. These can be updated in the Docker file. 
+
+All these steps are co-ordinated using AWS Step functions.
+
+# Steps
+After the perf-test stack has been deployed successfully, 
+
+1. Update the following ENV variables in the Dockerfile (both under `gatling-scripts` and `mock-scripts` folders) using the outputs from perf-test stack
+
+    ```
+    ENV COGNITO_CLIENT_ID <client_id>
```

Comment:
Cognito, GraphQL URL, etc. you can make a call to Param Store as we're storing those there.

https://github.com/aws-samples/aws-serverless-airline-booking/blob/develop/amplify.yml#L31

Reply:
I think having a bootstrap.sh, bootstrap.py, whatever language you want will be easier - There's a lot of moving parts, placeholders that people need to fill in, and a script can automate most of that.